kyo-bench/src/main/scala/kyo/bench/HttpClientRaceContentionBench.scala

```diff
@@ -0,0 +1,71 @@
+package kyo.bench
+
+import org.http4s.ember.client.EmberClientBuilder
+
+class HttpClientRaceContentionBench
+    extends Bench.ForkOnly("pong"):
+
+    val port        = 9999
+    val concurrency = 100
```

Comment:
We can't use 10k like the scenario here since the benchmark would take too long.